Makefile

```diff
@@ -189,6 +190,16 @@ registry-load-bundles: ## Load selected e2e testdata container images created in
 	testdata/bundles/registry-v1/build-push-e2e-bundle.sh ${E2E_REGISTRY_NAMESPACE} $(REGISTRY_ROOT)/bundles/registry-v1/prometheus-operator:v1.2.0 prometheus-operator.v1.2.0 prometheus-operator.v1.0.0
 	testdata/bundles/registry-v1/build-push-e2e-bundle.sh ${E2E_REGISTRY_NAMESPACE} $(REGISTRY_ROOT)/bundles/registry-v1/prometheus-operator:v2.0.0 prometheus-operator.v2.0.0 prometheus-operator.v1.0.0
 
+.PHONY: bootstrap-local-dev
+bootstrap-local-dev: $(KIND) $(SETUP_ENVTEST) #HELP Copies the kubernetes service certificates and starts kubefwd
```

Comment:
Might want to make this an `#EXHELP` comment.

Reply:
I'm gonna drop this and put it in a script for now. We have tilt.